### ISSUE TITLE
Add Projectile Firing Capability?

### ISSUE THREAD

**Issue by calben** [calben/networked-third-person-player-controller#10]
The character will need to shoot projectiles for our game.
I think that could be implemented at this level.

Projectile firing in direction of the camera is something @monicau has already worked with before.
Do you think you'd be able to set something up again?

**Comment by monicau** [comment]
It was the other Monica that worked on projectile firing in direction of camera. @mpabin !

**Comment by calben** [comment]
Really?  Didn't you do it with the traversal powers?

**Comment by monicau** [comment]
Not exactly, I used raycasting and it makes the character lerp to the direction of the ray, which is wherever the mouse is pointing, which is not necessarily at the center of the screen where the camera is pointing at.  I think Monica already implemented shooting in direction of camera, it just wasn't added to the arena for the last meeting

**Comment by calben** [comment]
Great, cool!

**Comment by mpabin** [comment]
Yeah it was me who implemented it. It believe it was buggy when we tried to integrate it and i didn't have time to figure out why. I shall make sure it works this time!!
